examples/webgl_panorama_cube.html

```diff
@@ -73,9 +73,9 @@
 
 				}
 
-				const imageObj = new Image();
+				const loader = new THREE.ImageLoader();
 
-				imageObj.onload = function () {
+				loader.load( atlasImgUrl, ( imageObj ) => {
```

Comment:
I'd suggest this pattern. Also, the loader returns `image`.
```js
new THREE.ImageLoader()
	.load( atlasImgUrl, ( image ) => {

} );
```
Be aware of white space.

Reply:
This code is relatively simple, but I think it may affect the reading experience. 
What do you think? @Mugen87